### DISCUSSION TITLE
Animações / Microinterações

### DISCUSSION THREAD

**Discussion by Victor-HM** [diego3g/rsxp-2023#106] [Ideas]
Estive pensando em adicionar algumas micro interações com o gesture handle e o reanimated que já estão na aplicação. Seria um detalhe muito interessante em adicionar.

**Comment by Rodrigo001-dev** [comment]
@Victor-HM  Gostei da ideia, mas em quais situações voce identificou que poderia ter micro interações ?

**Reply by Victor-HM** [reply]
Não pretendo fazer algo muito complexo, como por exemplo: na tela de speakers quando o usuário segurar em algum dos cards, a sua caixa iria aumentar um pouco e quando soltar ele voltaria do tamanho normal. Na parte da navegação com tabs, a cada clique em alguma aba aconteceria a mesma coisa com o ícone, mas com apenas um clique em vez de segurar. Com certeza tem mais micro interações que podem ser utilizadas, mas na essência é isso.

**Reply by Rodrigo001-dev** [reply]
Show de bola
@diego3g @rodrigorgtic O que vocês acham ?

**Reply by orodrigogo** [reply]
A ideia é muito legal. Eu gosto da ideia de utilizar micro interações. Mas, eu tomaria cuidado para não utilizar por utilizar apenas. E utilizar com objetivo de entregar algum tipo de funcionalidade ou tornar melhor a UX.

Acho que podemos esperar a primeira versão e ver quais funcionalidades poderiam ter uma experiência melhor para o usuário com micro interações. 💜
